ui/components/Notification/MesheryNotification.js

```diff
@@ -0,0 +1,182 @@
+import react, {useState} from "react";
+import { styled } from "@mui/material/styles";
+import {
+    IconButton,
+    Typography,
+    Divider,
+    Tooltip,
+    Badge,
+    Drawer,
+    Tabs,
+    Tab,
+  } from "@mui/material";
+  import NotificationsIcon from '@mui/icons-material/Notifications';
+  import DoneAllIcon from '@mui/icons-material/DoneAll';
+  import MesheryEventViewer from "./MesheryEventViewer";
+
+   
+  function MesheryNotification () {
+
+  const DrawerButton = styled(IconButton)((theme) => ({
+    padding : '0.45rem',
+    margin : '0.2rem',
+    backgroundColor : '#000000',
+    color : '#FFFFFF',
+    "&:hover" : { backgroundColor : '#FFFFFF',
+      color : '#000000' }
+  }))
+
+    const listTop = styled('div')(() => ({
+      display : 'grid',
+    alignItems : 'center',
+    gridTemplateColumns : "2fr 6fr 2fr",
+    paddingTop : theme.spacing(2),
+    paddingLeft : theme.spacing(1),
+    paddingRight : theme.spacing(1),
+    paddingBottom : theme.spacing(2),
+
+    }))
+
+    const NotifSelector = styled('div')(() => ({
+      display : 'flex', 
+
+    }))
+    const ClearAllButton = styled('div')(() => ({
+      display : 'flex', 
+      justifyContent : 'flex-end',
+    }))
+    
+    const [events, setEvents] = useState([]);
+    const [open, setOpen] = useState(false);
+    const [tabValue, setTabValue] = useState(0);
+    const [displayEventType, setDisplayEventType] = useState('*');
+
+    const handleToggle = () => {
+        setOpen(true);
+      };
+    const handleClose = () => {
+        setOpen(false);
+      };
+    
+    const handleNotifFiltering = (type) => {
+      // setDisplayEventType(type);
+    }  
+
+    const handleTabChange = ( newTabValue) => {
+      setTabValue(newTabValue) ;
+    }
+  
+    function getNotifications(events, type) {
+      if (!Array.isArray(events)) return [];
+    
+      if (type === "error") return events.filter(ev => ev.event_type === 2);
+      if (type === "warning") return events.filter(ev => ev.event_type === 1)
+      if (type === "success") return events.filter(ev => ev.event_type === 0)
+    
+      return events;
+    }
+
+    function getNotificationCount(events) {
+        if (!Array.isArray(events)) return 0;
+      
+        const errorEventCount = events.filter(ev => ev.event_type === 2).length;
+        const totalEventsCount = events.length;
+      
+        return errorEventCount || totalEventsCount;
+      }
+
+    return(
+        <div>
+        <IconButton color="inherit" size="large"  onClick={handleToggle}>
+        <Badge badgeContent={getNotificationCount(events)} >
+                <NotificationsIcon   />
+                </Badge>  
+            </IconButton>
+                   
+             <Drawer
+             sx={{
```

Comment:
again, think if this sort of styling can be reused and if so, then make it a reusable component.